lib/srv/desktop/rdp/rdpclient/client.go

```diff
@@ -519,26 +524,35 @@ func (c *Client) sharedDirectoryInfoRequest(req tdp.SharedDirectoryInfoRequest)
 	return C.ErrCodeSuccess
 }
 
-// close frees the memory of the cgo.Handle,
-// closes the RDP client connection,
-// and frees the Rust client.
+// close closes the RDP client connection and
+// the TDP connection to the browser.
 func (c *Client) close() {
 	c.closeOnce.Do(func() {
 		// Ensure the RDP connection is closed
 		if errCode := C.close_rdp(c.rustClient); errCode != C.ErrCodeSuccess {
 			c.cfg.Log.Warningf("error closing the RDP connection")
 		}
+		c.cfg.Log.Debug(("RDP connection closed successfully"))
+
+		// Ensure the TDP connection is closed
+		if err := c.cfg.Conn.Close(); err != nil {
+			c.cfg.Log.Warningf("error closing the TDP connection: %v", err)
+		}
+		c.cfg.Log.Debug(("TDP connection closed successfully"))
+	})
+}
 
+// cleanup frees the Rust client and
+// frees the memory of the cgo.Handle.
+func (c *Client) cleanup() {
+	c.cleanupOnce.Do(func() {
 		// Let the Rust side free its data
 		C.free_rdp(c.rustClient)
+		c.cfg.Log.Debug(("Rust client memory freed"))
```

Comment:
Can we remove this one? Looks like more of dev troubleshooting, not necessarily useful in the debug logs.